cmd/send-email/main.go

```diff
@@ -4,12 +4,14 @@ import (
     "github.com/alecthomas/kong"
 
     "github.com/edulinq/autograder/config"
+    "github.com/edulinq/autograder/db"
     "github.com/edulinq/autograder/email"
     "github.com/edulinq/autograder/log"
 )
 
 var args struct {
     config.ConfigArgs
+    Course string `help:"Optional Course ID. The course is only required when emailing by role or the *, which denotes all roles." arg:"" optional:""`
```

Comment:
The help string is reading a little awkward.
Maybe something like:
"Optional Course ID. Only required when roles or * (all course users) is are in the recipients."